talent-challenge-program2021/selected-projects.md

```diff
@@ -0,0 +1,43 @@
+## Selected Projects in Sprint 1
+
+Selected Projects in Sprint 1 are listed below. Mentee candidates who want to participate in a certain project please refer to [Mentee Selection Process](README.md#mentees)
+
+The difficulty of the project is divided into three levels: Very hard, Hard, and Medium, the corresponding pre-tax bonus is RMB 10000 to a very hard project, RMB 8000 to a hard project and RMB 5000 to a medium project respectively. 
+
+### Template
+
+```
+## TiDB Ecosystem Project Name
+
+## Title
+
+## Description
+
+what's the problem, how to solve it
+
+## Document Collection
+
+- Proposal doc: ${proposal doc}
+- Weekly report: ${weekly report link}
+
+## Talent Challenge Program information
+
+- Mentor of this issue: @github_id
+- Recommended skills: ${skills and URL link}
+- Estimated Workloads: (XL, man-month)
+
+## Milestones and action items
+
+Milestone 1: ${milestone descption}, Expected finish date: ${date}
+
+- [ ] action item 1
+- [ ] action item 2
+
+Milestone 2: ${milestone descption}, Expected finish date: ${date}
+
+- [ ] action item 1
+- [ ] action item 2
+
+```
```

Comment:
no need to describe the template again

Reply:
I think it is better to leave the template here. When each mentor upload their project, the template can be a convenient reference.